package/MDAnalysis/coordinates/memory.py

```diff
@@ -515,6 +515,10 @@ def timeseries(self, asel=None, start=0, stop=-1, step=1, order='afc'):
         .. versionchanged:: 1.0.0
            Deprecated `format` keyword has been removed. Use `order` instead.
         """
+        warnings.warn("MemoryReader.timeseries inclusive `start` and `stop`"
```

Comment:
Can  we do the warning only if `stop` is changed and not the default? 

The warning should only be about `start` becoming exclusive instead of inclusive.

Reply:
@orbeckst can warn only if stop is set. 

I assume you mean stop rather than start in "The warning should only be about start becoming exclusive instead of inclusive"?

Reply:
Yes, you’re right, of course. Sorry for the typo.